src/polygonFeature.js

```diff
@@ -67,7 +92,9 @@ var polygonFeature = function (arg) {
   /**
    * Get the internal coordinates whenever the data changes.  For now, we do
    * the computation in world coordinates, but we will need to work in GCS
-   * for other projections.
+   * for other projections.  Also compute the extents of the outside of each
+   * polygon for faster chcking if points are in the polygon.
```

Comment:
@manthey you may want to run a spell checker on code comments ``chcking --> checking``